test/augmentation/test_augmentation_3d.py

```diff
@@ -0,0 +1,748 @@
+                                 [0., 0., 0.],
+                                 [0., 1., 1.]]]]])  # 1 x 1 x 4 x 4
+        input = input.to(device)
+
+        expected = torch.tensor([[[[[0., 0., 0.],
+                                    [0., 0., 0.],
+                                    [1., 1., 0.]]]]])  # 1 x 1 x 4 x 4
+        expected = expected.to(device)
+
+        expected_transform = torch.tensor([[[-1., 0., 0., 3.],
+                                            [0., 1., 0., 0.],
+                                            [0., 0., 1., 0.],
+                                            [0., 0., 0., 1.]]])  # 1 x 4 x 4
+        expected_transform = expected_transform.to(device)
+
+        identity = torch.tensor([[[1., 0., 0., 0.],
+                                  [0., 1., 0., 0.],
+                                  [0., 0., 1., 0.],
+                                  [0., 0., 0., 1.]]])  # 1 x 4 x 4
+        identity = identity.to(device)
+
+        input = input.repeat(5, 3, 1, 1, 1)  # 5 x 4 x 4 x 3
+        expected = expected.repeat(5, 3, 1, 1, 1)  # 5 x 4 x 4 x 3
+        expected_transform = expected_transform.repeat(5, 1, 1)  # 5 x 4 x 4
+        identity = identity.repeat(5, 1, 1)  # 5 x 4 x 4
+
+        assert (f(input)[0] == expected).all()
+        assert (f(input)[1] == expected_transform).all()
+        assert (f1(input)[0] == input).all()
+        assert (f1(input)[1] == identity).all()
+
+    def test_same_on_batch(self, device):
+        f = RandomHorizontalFlip3D(p=0.5, same_on_batch=True)
+        input = torch.eye(3).unsqueeze(dim=0).unsqueeze(dim=0).repeat(2, 1, 1, 1, 1)
+        res = f(input)
+        assert (res[0] == res[1]).all()
+
+    def test_sequential(self, device):
+
+        f = nn.Sequential(
+            RandomHorizontalFlip3D(1.0, return_transform=True),
+            RandomHorizontalFlip3D(1.0, return_transform=True),
+        )
+        f1 = nn.Sequential(
+            RandomHorizontalFlip3D(1.0, return_transform=True),
+            RandomHorizontalFlip3D(1.0),
+        )
+
+        input = torch.tensor([[[[0., 0., 0.],
+                                [0., 0., 0.],
+                                [0., 1., 1.]]]])  # 1 x 1 x 3 x 3
+        input = input.to(device)
+
+        expected_transform = torch.tensor([[[-1., 0., 0., 3.],
+                                            [0., 1., 0., 0.],
+                                            [0., 0., 1., 0.],
+                                            [0., 0., 0., 1.]]])  # 1 x 4 x 4
+        expected_transform = expected_transform.to(device)
+
+        expected_transform_1 = expected_transform @ expected_transform
+        expected_transform_1 = expected_transform_1.to(device)
+
+        assert(f(input)[0] == input).all()
+        assert(f(input)[1] == expected_transform_1).all()
+        assert(f1(input)[0] == input).all()
+        assert(f1(input)[1] == expected_transform).all()
+
+    @pytest.mark.skip(reason="turn off all jit for a while")
```

Comment:
we should start enabling jit test in the next release. Will add as item in the roadmap